ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ScanConfigWizardFooter.tsx

```diff
@@ -38,7 +38,7 @@ function ScanConfigWizardFooter({
                     </Button>
                 ) : (
                     <Button variant="primary" type="submit" onClick={onSave} isLoading={isSaving}>
```

Comment:
Nit: it would be nice to disable this button during saving as well, so that users don't double-submit the form.